spec/models/user_spec.rb

```diff
@@ -58,15 +58,15 @@
         Timecop.return
       end
 
-      it 'returns 2 counts even though current time is Nov 1st' do
+      it 'returns 0 counts because current time is Nov 1st' do
```

Comment:
Given system time is Oct 31rd, for a user in Australia `count_hits` method will return 0 hits because the quota should have been reset from the user's perspective.

So this particular spec tells us that changes introduced here in the commit indeed fix the issue with the time_zone.